src/exchange/group/accounts/group-accounts.controller.js

```diff
@@ -79,17 +92,53 @@ angular.module('Module.exchange.controllers').controller(
       }
     }
 
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
+      return {
+        displayName: name,
+        name: name
```

Comment:
You will ended up with an ESLint `object-shorthand` and `comma-dangle` error.
```suggestion
        name,
```